src/modules/mavlink/mavlink_ftp.cpp

```diff
@@ -971,6 +971,16 @@ void MavlinkFTP::send(const hrt_abstime t)
 				_work_buffer2 = nullptr;
 			}
 		}
+
+	} else if (_session_info.fd != -1) {
+		// close session without activity
+		if (hrt_elapsed_time(&_last_work_buffer_access) > 10000000) {
```

Comment:
```suggestion
		if (hrt_elapsed_time(&_last_work_buffer_access) > 10_s) {
```
And add `using namespace time_literals;` at the top of the file.